openapi.yaml

```diff
@@ -2171,6 +2171,11 @@ components:
         realTime:
           description: Whether there is real-time data about this leg
           type: boolean
+        scheduled:
```

Comment:
maybe an additional field for replacement would make sense to reference the original trip?

Reply:
as long as we don't have to change the API for this, that's fine :)